.github/workflows/template-sync.yml

```
@@ -7,18 +7,24 @@
+          token: ${{ steps.generate_token.outputs.token }}
 
       - name: actions-template-sync
-        uses: AndreasAugustin/actions-template-sync@v2.2.3
+        uses: AndreasAugustin/actions-template-sync@v2
         with:
-          github_token: ${{ secrets.GITHUB_TOKEN }}
+          github_token: ${{ steps.generate_token.outputs.token }}
           source_repo_path: VilnaCRM-Org/php-service-template
           upstream_branch: main
-          pr_labels: template_sync
+          pr_labels: template_sync
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)